search-service/search-infra/src/main/kotlin/com/grepp/quizy/search/infra/quiz/repository/UserAnswerElasticRepository.kt

```diff
@@ -0,0 +1,11 @@
+package com.grepp.quizy.search.infra.quiz.repository
+
+import com.grepp.quizy.search.infra.quiz.document.UserAnswerDocument
+import org.springframework.data.elasticsearch.repository.ElasticsearchRepository
+
+interface UserAnswerElasticRepository : ElasticsearchRepository<UserAnswerDocument, Long> {
+
+    fun getByUserId(id: Long): UserAnswerDocument? {
+        return findById(id).orElse(null)
```

Comment:
사실 이거 남기러왔는데 여기도 되는지 모르겠는데
`findByIdOrNull`이라는게 가능합니다

```suggestion
        return findByIdOrNull(id)
```
이렇게 하면 코드를 줄일 수 있을 것 같아요 !

Reply:
오 대박 저건 몰랐습니다 감사합니다!